lib/tasks/development.rb

```diff
@@ -57,7 +57,9 @@ def podman_rt_args
     [
       '--security-opt', 'label=disable',
       "--uidmap=#{user.uid}:0:1", "--uidmap=0:1:#{user.uid}",
-      "--gidmap=#{user.gid}:0:1", "--gidmap=0:1:#{user.gid}"
+      "--gidmap=#{user.gid}:0:1", "--gidmap=0:1:#{user.gid}",
+      *("--uidmap=#{user.uid+1}:#{user.uid+1}:1000" if user.uid < 1000),
+      *("--gidmap=#{user.gid+1}:#{user.gid+1}:1000" if user.gid < 1000),
     ].tap do |arr|
```

Comment:
Do we need to `compact` here to get rid of `nil`s?  I've never seen the `*` operator used like this.

Reply:
The `*()` takes care of the `nil`s.
```
3.1.2 :004 > [*(nil)]
 => []
```
I can remove the `*` operator and add `compact´ if that makes things clearer.

Reply:
OK! Yea let's use compact just to be more uniform with the rest of the code base.

Reply:
Fixed.